documentation/markup.rst

```diff
@@ -565,6 +567,11 @@ The directives are:
 
             Description of the attribute.
 
+
+   Refer to an attribute, use the ``:attr:`` role::
```

Comment:
```suggestion
   Refer to an attribute using the ``:attr:`` role::
```

Reply:
While this is useful here, the same could be done for other directives too, like the `class` just above or the `method` just below.  It would be nice if the documentation was consistent.

Reply:
> While this is useful here, the same could be done for other directives too, like the `class` just above or the `method` just below. It would be nice if the documentation was consistent.

should it be seperate pr?

Reply:
Either way is fine, but smaller PRs are easier to review and merge.

Would you like to make a separate follow-up PR? :)

Reply:
> Either way is fine, but smaller PRs are easier to review and merge.
> 
> Would you like to make a separate follow-up PR? :)

sure. and when will this pr be merged ? #1311

Reply:
Let's do it now :)

Thank you for the PR!